tests/sql_repository_test.py

```diff
@@ -0,0 +1,89 @@
+from faker import Faker
+
+fake = Faker()
+
+existing_elements_registry = []
+last_deleted_element = None
+
+
+def test_create(sql_repository):
+    """Should create a new Entry"""
```

Comment:
why not naming the function/method name as 

`test_creates_a_new_entry`

Then you can get rid of comments

Reply:
This function just tests the function create of the repository, this is a simple and unique name. As I explained before the comment in the function is providing a verbose description of the test.